### DISCUSSION TITLE
How to reload all the data into read servers when using kubernetes ??

### DISCUSSION THREAD

**Discussion by dixonhsiao** [milvus-io/milvus#4287] [Q&A and General discussion]
For example, I have inserted 8GB of data that seemed distributed in 3 read servers (3+3+2 = 8GB). But when I remove the milvus workloads and services in kubernetes and redeploy them again, it seems that it only loads about 2.27GB, which only occupied one read server. And the problem is the search time becomes much slower, something that I can feel about 2~3 sec for each search, but the search time is much faster before the redeployment, when 8GB data is distributed in 3 read servers...like 0.0? sec for each search...  so my question is : is it possible to reload all the data into read servers when redeployment ?? or is it designed to just reload the index ??

**Comment by miia12** [comment]
- Are you using mishards for read/write separation? 
- What‘s the version of mishards?
- What's the configuration of each node?
